backend/lib/endpoints/schema/users.js

```diff
@@ -5,6 +5,19 @@ const getUserByIdSchema = {
   params: strictSchema().prop("userId", S.string().required()),
 };
 
+const createProfileSchema = {
```

Comment:
to stay consistent with the other files, schema's etc. (and API route /users) I think this should probably be named `createUserSchema`